src/libraries/System.Net.Http/src/System/Net/Http/HttpVersionPolicy.cs

```diff
@@ -0,0 +1,12 @@
+// Licensed to the .NET Foundation under one or more agreements.
+// The .NET Foundation licenses this file to you under the MIT license.
+
+namespace System.Net.Http
+{
+    public enum HttpVersionPolicy
+    {
+        RequestVersionOrLower,
+        RequestVersionOrHigher,
+        RequestVersionExact
```

Comment:
The enum and its values all need XML comments for the docs.

Reply:
Added.